koku/masu/openapi.json

```diff
@@ -1059,6 +1063,38 @@
         ]
       }
     },
+    "/clear_celery_queues/":{
+      "get":{
+        "summary": "Returns the number of tasks cleared the celery queues.",
+        "operationId":"clearCeleryQueues",
+        "description": "Returns the number of tasks cleared the celery queues.",
+        "parameters":[
+          {
+            "name": "clear_all",
+            "description":"whether to clear all celery queues.",
+            "in":"query",
+            "schema":{
+              "type":"boolean"
+            }
+          }
+        ],
+        "responses":{
+          "200":{
+            "description": "Returns the number of tasks cleared the celery queue.",
+            "content":{
+              "application/json":{
+                "schema":{
+                  "$ref":"#/components/schemas/clearCeleryQueue"
```

Comment:
You will need to create this ref later down in the code. You can use `#/components/schemas/celeryQueueLength` as example. 

We usually use the swagger editor online to confirm openapi.json changes: https://editor.swagger.io/

Reply:
thanks for pointing out swagger 👍🏿